__tests__/app.test.js

```diff
@@ -127,5 +127,41 @@ describe('app.js test suite', () => {
                 
                 expect(msg).toBe('There are no comments for this article or no such article exists')
 
+            })
         })
-});
\ No newline at end of file
+    })
+
+    describe('PATCH /api/articles/:article_id', () => {
+        test('increments the votes for an article and responds with the updated article object', () => {
```

Comment:
Please also test for:
Happy path - check that the votes have been committed to the database - utilise your GET articles by id endpoint and assert that you expect the votes value to be the new amount after the patch.

Sad path - Please test for article id not found and article id wrong data type.